SharedUtilities.gs

```diff
@@ -49,7 +49,7 @@ let REQUEST_SUBMISSION_EMAIL_TEMPLATE = `
 Dear {AmbassadorName},
 Please submit your deliverables for {Month} {Year} using the link below:
 {SubmissionFormURL}
-The deadline is {Deadline} days.
+The deadline is {SUBMISSION_WINDOW_MINUTES} days.
```

Comment:
This text isn't very clear - it comes out like this: 
`The deadline is 7 days.`

It would be better to do the date math and tell them the deadline date, not just 7 days. If that is difficult, I can get over it.

Reply:
this already fixed in last version